PangleAdapter/src/main/java/com/chartboost/mediation/pangleadapter/PangleAdapter.kt

```diff
@@ -337,6 +343,7 @@ class PangleAdapter : PartnerAdapter {
                 PartnerLogController.log(INVALIDATE_SUCCEEDED)
                 Result.success(partnerAd)
             }
+
```

Comment:
can you re-verify pangle still doesn't have destroy for fullscreen?

Reply:
Not that am aware of, or found in their docs. Let me do another run through.

Reply:
Yep, no `destroy` method in their fullscreen ads. 👍🏼